EasyCommands.Tests/EasyCommands.Tests.csproj

```diff
@@ -173,11 +173,16 @@
     <Compile Include="ScriptTests\FunctionalTests\ControlCommandTests.cs" />
     <Compile Include="ScriptTests\FunctionalTests\IterationCommandTests.cs" />
     <Compile Include="ScriptTests\FunctionalTests\SimpleWaitCommandTests.cs" />
-    <Compile Include="ScriptTests\FunctionalTests\SimpleComparisonTests.cs" />
+    <Compile Include="ScriptTests\FunctionalTests\SimpleComparisonsOtherTests.cs" />
     <Compile Include="ScriptTests\FunctionalTests\SimpleAggregationTests.cs" />
     <Compile Include="ScriptTests\FunctionalTests\SimpleColorTests.cs" />
     <Compile Include="ScriptTests\FunctionalTests\SimpleVectorTests.cs" />
     <Compile Include="ScriptTests\FunctionalTests\SimpleCastTests.cs" />
+    <Compile Include="ScriptTests\FunctionalTests\SimpleComparisonsBooleanTests.cs" />
+    <Compile Include="ScriptTests\FunctionalTests\SimpleComparisonsNumberTests.cs" />
+    <Compile Include="ScriptTests\FunctionalTests\SimpleComparisonsStringTests.cs" />
+    <Compile Include="ScriptTests\FunctionalTests\SimpleComparisonsVectorTests.cs" />
+    <Compile Include="ScriptTests\FunctionalTests\SimpleComparisonsListTests.cs" />
```

Comment:
Looks like we need to add Color Tests.

Reply:
the color comparison are in the "other" test since they are incomplete (only equality/inequality).
I have no idea what sense the other comparisons would make.

Reply:
I see.  I need to re-do the structure of the tests and introduce "Operations" tests, with "comparison" folder containing all of these comparison tests.  Obviously that'll be a separate effort.